depthimage_to_pointcloud2/src/depthimage_to_pointcloud2_node.cpp

```diff
@@ -64,7 +65,7 @@ static void depthCb(const sensor_msgs::msg::Image::SharedPtr image)
   } else if (image->encoding == sensor_msgs::image_encodings::TYPE_32FC1) {
     depthimage_to_pointcloud2::convert<float>(image, cloud_msg, model);
   } else {
-    fprintf(stderr, "Depth image has unsupported encoding [%s]\n", image->encoding.c_str());
+    RCUTILS_LOG_WARN("Depth image has unsupported encoding [%s]", image->encoding.c_str())
```

Comment:
We could use a throttled logging here to match the original code.

Reply:
Will change this.